.gitattributes

```diff
@@ -0,0 +1 @@
+*.html linguist-language = Java
```

Comment:
I think that mimicking all HTML files as Java source code is not a very good idea. According to the Linguist [documentation](https://github.com/github/linguist#documentation), the following line looks like a more correct way:

```
gson/docs/javadocs/* linguist-documentation
```